### PR TITLE
Add OpenEBS v0.8 Chart

### PR DESCRIPTION
**Pull request by kmova**
OpenEBS 0.8 add support for:
- cStor Storage Engine
- Further enhancements to Jiva Storage Engine
- Ability to send anonymous usage statistics
- More flexible Storage Policy declarations for Storage Engines
  via CAS Templates
- Automated setting up of default storage classes.

This release also removes the bundling of Prometheus and Grafana
and is limited to installing only OpenEBS core components.

Signed-off-by: kmova <kiran.mova@openebs.io>

### CODE REVIEW

**Comment by kmova** [comment]
Hi @guangbochen, I was wondering if you would have time to review this PR or could you point me to someone who can help me with the reviews.

**Comment by guangbochen** [comment]
I will review this PR thanks, can we rename the folder from `v0.8` to `latest`, it will be easy for us to track the commit changes in the future, thanks.

**Comment by kmova** [comment]
Thank you @guangbochen. Appreciate your quick response and help!
Pushed a commit by renaming v0.8 to latest.

File: proposed/openebs/latest/questions.yml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    label: Image Tag For OpenEBS Exporter Image
+    show_if: "policies.monitoring.enabled=true&&defaultImage=false"
+- variable: ndm.filters.excludeVendors
+  default: "CLOUDBYT,OpenEBS"
+  type: string
+  description: "Configure NDM to filter disks from following vendors"
+  label: Filter Disks belonging to vendors
+  group: "NDM Disk Filter by Vendor "
+- variable: ndm.filters.excludePaths
+  default: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md"
```
Comment (guangbochen) [inline proposed/openebs/latest/questions.yml]: need to channge to `loop\,fd0\,sr0\,/dev/ram\,/dev/dm-\,/dev/md`

File: proposed/openebs/latest/questions.yml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    label: Monitoring Exporter Image
+    show_if: "policies.monitoring.enabled=true&&defaultImage=false"
+  - variable: policies.monitoring.imageTag
+    default: "0.8.0"
+    description: "The image tag of OpenEBS Exporter"
+    type: string
+    label: Image Tag For OpenEBS Exporter Image
+    show_if: "policies.monitoring.enabled=true&&defaultImage=false"
+- variable: ndm.filters.excludeVendors
+  default: "CLOUDBYT,OpenEBS"
```
Comment (guangbochen) [inline proposed/openebs/latest/questions.yml]: need to channge to `CLOUDBYT\,OpenEBS`

**Comment by guangbochen** [comment]
@kmova just need to address the above two comments and I will merge this PR, thanks.

**Comment by kmova** [comment]
Thanks @guangbochen addressed the comments.

File: proposed/openebs/latest/questions.yml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    label: Monitoring Exporter Image
+    show_if: "policies.monitoring.enabled=true&&defaultImage=false"
+  - variable: policies.monitoring.imageTag
+    default: "0.8.0"
+    description: "The image tag of OpenEBS Exporter"
+    type: string
+    label: Image Tag For OpenEBS Exporter Image
+    show_if: "policies.monitoring.enabled=true&&defaultImage=false"
+- variable: ndm.filters.excludeVendors
+  default: "CLOUDBYT\,OpenEBS"
```
Comment (guangbochen) [inline proposed/openebs/latest/questions.yml]: need to using single quotation as 'CLOUDBYT\,OpenEBS'

File: proposed/openebs/latest/questions.yml [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  type: string
+  description: "Configure NDM to filter disks from following vendors"
+  label: Filter Disks belonging to vendors
+  group: "NDM Disk Filter by Vendor "
+- variable: ndm.filters.excludePaths
+  default: "loop\,fd0\,sr0\,/dev/ram\,/dev/dm-\,/dev/md"
```
Comment (guangbochen) [inline proposed/openebs/latest/questions.yml]: same here for single quotation.

**Comment by kmova** [comment]
Sorry about that @guangbochen. Updated now.

**Comment by guangbochen** [comment]
LGTM.

**Comment by kmova** [comment]
Thank you @guangbochen . Appreciate your help with reviewing this PR.

**Pull request merged**